test/SlashCommands-test.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { MatrixClient } from 'matrix-js-sdk/src/matrix';
+
+import { getCommand } from '../src/SlashCommands';
+import { createTestClient } from './test-utils';
+import { MatrixClientPeg } from '../src/MatrixClientPeg';
+
+describe('SlashCommands', () => {
+    let client: MatrixClient;
+
+    beforeEach(() => {
+        client = createTestClient();
+        jest.spyOn(MatrixClientPeg, 'get').mockReturnValue(client);
+    });
+
+    describe('/topic', () => {
+        it('sets topic', async () => {
+            const command = getCommand("/topic pizza");
+            expect(command.cmd).toBeDefined();
```

Comment:
should we be checking that `command` itself is defined too?

Reply:
Looks like it'll always return an object: https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/SlashCommands.tsx#L1351

Reply:
the jsdoc doesn't match on that function - can we fix that while we're here?

Reply:
Oh wow, good catch!